### DISCUSSION TITLE
PLS add a way to turn on dev mode in the chrome ext page

### DISCUSSION THREAD

**Discussion by poooooooper** [3kh0/ext-remover#470] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
i wish this title of the question would come true

**Comment by fnfgo** [comment]
Get a new chromebook thats not managed

**Comment by rwg100** [comment]
why is bro on github at 1 in the morning

**Reply by fnfgo** [reply]
who?

**Reply by BroJac5246** [reply]
A lot of us have different time zones.

**Reply by Aka-but-nice** [reply]
True, right now it is 10:26 am.
